src/safeds/ml/nn/_output_conversion_time_series.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from torch import Tensor
+from safeds.data.labeled.containers import TimeSeriesDataset
+from safeds.data.tabular.containers import Column, Table
+from safeds.ml.nn._output_conversion import OutputConversion
+
+
+class OutputConversionTimeSeries(OutputConversion[TimeSeriesDataset, TimeSeriesDataset]):
+    """The output conversion for a neural network, defines the output parameters for the neural network."""
+
+    def __init__(self, prediction_name: str = "prediction_nn", window_size: int = 1, forecast_horizon: int = 1) -> None:
+        """
+        Define the output parameters for the neural network in the output conversion.
+
+        Parameters
+        ----------
+        prediction_name
+            The name of the new column where the prediction will be stored.
+        """
+        self._prediction_name = prediction_name
+        self._window_size = window_size
+        self._forecast_horizon = forecast_horizon
```

Comment:
Are those always the same as in the `ÌnputConversionTimeSeries`? If so, you could automatically give the `OutputConversionTimeSeries` these parameters.

Reply:
Yes they are, adding it